07LessonSeven/in-class-project/css/my-style.css

```diff
@@ -0,0 +1,87 @@
+#container {
+  width:850px;
+  background-image:url(../img/nasa-background.jpg);
+  height:450px;
+  position: relative;
+  margin: auto;
+  margin-bottom:20px;
+
+}
+
+#container h1 {
+  position: absolute;
+  text-align: center;
+  font-family:sans-serif;
+  color:white;
+  font-size: 80px;
+  top:40%;
+  width:100%;
+  margin:0;
+ }
+
+ #graybox {
+   width:850px;
+   height:324px;
+   background-color:#dddddd;
+   margin:auto;
+ }
+
+.smallboxes {
+  width:270px;
+  height:324px;
+  float:left;
+  text-align: left;
+  font-weight: bolder;
+  font-family: sans-serif;
+  padding:10px 30px;
+  box-sizing: border-box;
+}
+
+.smallboxes h5 {
+  font-weight: lighter;
```

Comment:
font weight should be in numeric values, like 100 here for lighter weights. they increment by 100s. 700 is bold!